src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -671,6 +672,10 @@ protected void buildShapelessRecipes(Consumer<IFinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(MSBlocks.CHISELED_GREEN_STONE_BRICKS).grist(BUILD, 2).grist(CAULK, 1).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSBlocks.HORIZONTAL_GREEN_STONE_BRICKS).grist(BUILD, 2).grist(CAULK, 1).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSBlocks.VERTICAL_GREEN_STONE_BRICKS).grist(BUILD, 2).grist(CAULK, 1).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.CHISELED_SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.RED_SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSBlocks.CHISELED_RED_SANDSTONE_COLUMN).grist(BUILD, 4).build(recipeSaver);
```

Comment:
I think it makes more sense to leave the grist cost to the grist cost generator to figure out their cost. You're adding regular crafting recipes for the columns, and the grist cost generator should be able to figure out a cost from that.

Reply:
Then as soon as the lines are deleted and tested, I consider this ready to merge

Reply:
Just tested it and it works without it for the sandstone columns, but the pink stone columns become unalchemizable.

Reply:
Then just for the moment, keep the pink, remove the sandstone, and youre golden

Reply:
Okay, done.